app/models/customer.rb

```diff
@@ -0,0 +1,5 @@
+class Customer < ApplicationRecord
+  has_many :services
```

Comment:
At this point, I haven't had a need to delete a customer. Is this something that you predict needing in the future and so you add it? Or do you add it when you need it?

Reply:
It's just generally good practice to have a dependent destroy when you have a foreign key constraint because the records are not allowed to exist without their parent. Really up to you, but just what I've picked up in DBA thus far